src/StreamFactoryInterface.php

```diff
@@ -48,5 +16,5 @@ public function createStreamFromResource($body);
      *
      * @return StreamInterface
      */
-    public function createStreamFromString($body);
+    public function createStream($body = '');
```

Comment:
> See you cannot make use the second parameter which is useful.

In my opinion, this does not matter at all. However, I do think the default should be to create a stream from a resource only. That way there is no concern about not using this parameter and it works better in all situations, as the stream does not need to be created by the factory.